controllers/nginx/configuration.md

```diff
@@ -434,6 +434,8 @@ The default mime type list to compress is: `application/atom+xml application/jav
 
 **add-headers:** Sets custom headers from a configmap before sending traffic to the client. See `proxy-set-headers` [example](https://github.com/kubernetes/ingress/tree/master/examples/customization/custom-headers/nginx)
 
+**bind-address:** Sets the addresses on which the server will accept requests instead of *.
+
```

Comment:
Please add a note that hostNetwork is required to work or the controller will crash loop